File: MODUL 4 Praktikum Pemrograman/PRAKTIKUM 4 C/PRAK405-2410817120021-Aulia-Az-Zahra.c
#include <stdio.h>
int main()
{
    int m, n;
    scanf("%d %d", &m, &n);
    int all_result = 0;
    for (int i = 1;i <= m; ++i)
    {
        int result = 0;
        printf("\n");
        for (int j = i; j >= 1; --j)
        {
            int term = j * n;
            result += term;
            all_result += term;
            printf("(%d * %d)", j, n);

            if (j > 1)
            {
                printf(" + ");
            }
        }
        printf(" = %d", result);
    }
        printf("\n%d", all_result);
    
    return 0;
}

File: MODUL 4 Praktikum Pemrograman/PRAKTIKUM 4 PY/PRAK405-2410817120021-Aulia-Az-Zahra.py
m, n = map(int,input().split())
all_result = 0
print()

for i in range(1,m + 1):
    result = 0

    for j in range(i, 0, -1):
        term = j * n
        result += term
        all_result += term
        print(f"({j} * {n})", end=" ")

        if j > 1:
            print("+", end=" ")
    print(f"= {result}", end=" ")
    print()
print(all_result)